### ISSUE TITLE
gnuarmeclipse-qemu-osx-2.6.0-201607280535.pkg missing liblzo shared library

### ISSUE THREAD

**Issue by bjandre** [gnuarmeclipse/qemu#27]
The macOS installer package for gnuarmeclipse-qemu-osx-2.6.0-201607280535 is missing the liblzo shared library:

```
$ /Applications/GNU\ ARM\ Eclipse/QEMU/2.6.0-201607280535/bin/qemu-system-gnuarmeclipse --version
dyld: Library not loaded: /opt/macports/lib/liblzo2.2.dylib
  Referenced from: /Applications/GNU ARM Eclipse/QEMU/2.6.0-201607280535/bin/qemu-system-gnuarmeclipse
  Reason: image not found
Trace/BPT trap: 5
```

liblzo2.2.dylib should probably be included in the libraries included in:
`/Applications/GNU\ ARM\ Eclipse/QEMU/2.6.0-201607280535/bin/`.

A work around is to install liblzo from another source. For example:

```
$ brew install lzo
...
...
$ /Applications/GNU\ ARM\ Eclipse/QEMU/2.6.0-201607280535/bin/qemu-system-gnuarmeclipse --version
GNU ARM Eclipse 64-bits QEMU emulator version 2.6.0
Copyright (c) 2003-2008 Fabrice Bellard
```

**Comment by ilg-ul** [comment]
thank you for the bug report, I'll fix it with the next release.

what os version are you using?

**Comment by bjandre** [comment]
I'm running:
- OS X El Capitan, v10.11.6.
- xcode 8.0 (8A218a)
- homebrew instead of macports.

**Comment by adamvn** [comment]
Had, same issue:
OSX, 10.12 and used homebrew to fix.

Thanks.

**Comment by ilg-ul** [comment]
I just fixed the build scripts (https://github.com/gnuarmeclipse/build-scripts), could you try to build your own qemu and let me know if the scripts and the resulting package are ok?

alternatively I have to send you a .pkg file, but I wouldn't make a new release only for this, I plan to upgrade to 2.7 soon.

**Comment by ilg-ul** [comment]
actually I added the binary files to the 2.6.0 release, check the ones ending in `-dev`.

**Comment by adamvn** [comment]
Hi thanks,

Busy trying to get my mac setup for build.. that seems much simpler.

-Adam

**Comment by adamvn** [comment]
great, -dev works just fine.

**Comment by ilg-ul** [comment]
please do not reply by mail to these messages, since this requires manual editing of the posts to remove the quoted parts.

**Comment by xprite** [comment]
This is my experience trying to run qemu on El Capitan 10.11.6 (15G31)

$ ./qemu-system-gnuarmeclipse --version
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /Applications/GNU ARM Eclipse/QEMU/2.6.0-201610170917-dev/bin/./qemu-system-gnuarmeclipse (which was built for Mac OS X 10.12)
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _clock_gettime
  Referenced from: /Applications/GNU ARM Eclipse/QEMU/2.6.0-201610170917-dev/bin/./qemu-system-gnuarmeclipse (which was built for Mac OS X 10.12)
  Expected in: /usr/lib/libSystem.B.dylib

Trace/BPT trap: 5

**Comment by ilg-ul** [comment]
hmmm... it looks like applications built for 10.12 are no longer running on 10.11.

unfortunately all my Macs are now at 10.12; do you know any trick to generate applications for 10.11?

**Comment by xprite** [comment]
I have no idea. I just updated to macOS Sierra and QEMU is now fully functional.

**Comment by ilg-ul** [comment]
> it looks like applications built for 10.12 are no longer running on 10.11.

unfortunately this is a know problem of Xcode 8 :-(

https://bugs.erlang.org/browse/ERL-256

**Comment by ilg-ul** [comment]
the current workaround seems to be to completely avoid `clock_gettime()` on macOS builds.

if you still have a 10.11 running somewhere, I can provide a binary.

**Comment by xprite** [comment]
I am good with 10.12 now. Maybe somebody else will be interested. Thank you

**Comment by ilg-ul** [comment]
fixed in v2.7.0-20161029
